flink-test-utils-parent/flink-test-utils/src/main/java/org/apache/flink/test/util/CollectionTestEnvironment.java

```diff
@@ -42,11 +42,6 @@ public JobExecutionResult getLastJobExecutionResult() {
 		}
 	}
 
-	@Override
-	public JobExecutionResult execute() throws Exception {
```

Comment:
IMO `execute()` should be a thin interface for adding default parameter, so we should mark it as `final` and this override makes non-sense.

Reply:
hmmm mark method as `final` breaks b/w comp. but I think it can be removed on the fly for these meaningless override.